### PR TITLE
fix: error screen

### PR DESCRIPTION
**Pull request by randhir-devtron**
# Description

Extract a InfoIconTippy component from TippyCustomized

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] Does this PR require documentation updates?
- [ ] I've updated documentation as required by this PR.
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Common/ErrorPage.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { useHistory } from 'react-router'
+import { DISCORD_LINK, ERROR_EMPTY_SCREEN } from './Constants'
+import GenericEmptyState from './EmptyState/GenericEmptyState'
+import { ErrorPageType } from './Types'
+import { noop } from './Helper'
+
+const ErrorPage = ({ code, image, title, subTitle, imageType }: ErrorPageType) => {
+    const { push } = useHistory()
+    const redirectToHome = () => {
+        push('/app/list')
```
Comment (shivani170) [inline src/Common/ErrorPage.tsx:10]: use constant for app/list

File: src/Common/ErrorPage.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+    const refresh = () => {
+        window.location.reload()
+    }
+    const reportIssue = () => {
+        window.open(DISCORD_LINK)
+    }
+    const getErrorPageProps = (statusCode: number): [() => void, string, boolean] => {
+        switch (statusCode) {
+            case 400:
```
Comment (shivani170) [inline src/Common/ErrorPage.tsx:20]: use enum for error code

File: src/Common/ErrorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import { ErrorPageType } from './Types'
+import { noop } from './Helper'
+
+const ErrorPage = ({ code, image, title, subTitle, imageType }: ErrorPageType) => {
+    const { push } = useHistory()
+    const redirectToHome = () => {
+        push('/app/list')
+    }
+    const refresh = () => {
+        window.location.reload()
```
Comment (shivani170) [inline src/Common/ErrorPage.tsx:13]: Make it a separate function

File: src/Common/Constants.ts [hunk 4]
```diff
@@ -44,22 +46,51 @@ export const DEFAULT_TAG_DATA = {
     isSuggested: true,
 }
 
+export enum API_STATUS_CODES {
```
Comment (shivani170) [inline src/Common/Constants.ts:49]: ERROR_STATUS_CODE

File: src/Common/EmptyState/GenericEmptyState.tsx [hunk 5]
```diff
@@ -63,12 +63,18 @@ const GenericEmptyState = ({
...
+                className={`flex column dc__gap-20 dc__mxw-300 ${
                     isRowLayout ? 'dc__align-start' : ''
                 } ${contentClassName}`}
             >
-                <h4 className="title fw-6 cn-9 mt-0 mb-0 lh-1-5">{title}</h4>
-                {subTitle && <p className={`subtitle ${isRowLayout ? 'subtitle--text-start' : ''}`}>{subTitle}</p>}
+                <div className="flex column dc__gap-8">
+                    <h4 className="title fw-6 cn-9 mt-0 mb-0 lh-24">{title}</h4>
+                    {subTitle && (
+                        <p className={`subtitle ${isRowLayout ? 'subtitle--text-start' : ''} ${subtitleClassName}`}>
```
Comment (shivani170) [inline src/Common/EmptyState/GenericEmptyState.tsx:73]: remove unused subTitle props

**Review by vivek-devtron — DISMISSED** [review]

**Review by shivani170 — APPROVED** [review]

**Pull request merged**